src/Umbraco.Web.UI.Client/gulp/config.js

```diff
@@ -28,9 +28,9 @@ module.exports = {
             installer: { files: "./src/installer/**/*.js", out: "umbraco.installer.js" },
             filters: { files: "./src/common/filters/**/*.js", out: "umbraco.filters.js" },
             resources: { files: "./src/common/resources/**/*.js", out: "umbraco.resources.js" },
-            services: { files: "./src/common/services/**/*.js", out: "umbraco.services.js" },
+            services: { files: ["./src/common/services/**/*.js", "./src/utility.js"], out: "umbraco.services.js" },
```

Comment:
@nathanwoulfe I may be a little picky, but as the global variable name is now in plural, shouldn't the file name also be? Both the actual file name and the reference here.

Reply:
@abjerner yup, I agree, naming should be consistent. Will fix that.

As for the _ vs Angular bits, I think we'll need to make those changes case-by-case, in some places it will be fine to swap, others, like you said, the different implementation may return unexpected results...

Reply:
@nathanwoulfe great 👍 

I think the copy function is the most critical. For various infinite editors and property editors, Umbraco will make a deep clone of the selection so the user makes changes to the copy rather than the original selection. So I think this is the most important one to have working as-is, as it *will* break existing functionality.

I don't know the impact of the other functions, but I suspect they are less important.